spring-grpc-spring-boot-autoconfigure/src/test/java/org/springframework/grpc/autoconfigure/server/GrpcServerReflectionAutoConfigurationTests.java

```diff
@@ -0,0 +1,34 @@
+package org.springframework.grpc.autoconfigure.server;
+
+import io.grpc.BindableService;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.autoconfigure.AutoConfigurations;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+import org.springframework.grpc.server.lifecycle.GrpcServerLifecycle;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GrpcServerReflectionAutoConfigurationTests {
+
+	private ApplicationContextRunner contextRunner() {
+		return new ApplicationContextRunner()
+			.withConfiguration(AutoConfigurations.of(GrpcServerReflectionAutoConfiguration.class))
+			.withBean("noopServerLifecylcle", GrpcServerLifecycle.class, Mockito::mock);
+	}
+
+	@Test
```

Comment:
For the "binary" flag properties I like to add 3 test cases...
1. property not set at all - what does the default do
2. property set to true
3. property set to false.

Wdyt?

Reply:
Done.